tests/integration/test_cross_model.py

```diff
@@ -14,9 +14,12 @@ def test_offer_and_consume(juju: jubilant.Juju, model2: jubilant.Juju):
     model2.integrate('dbalias', 'testapp')
 
     status = juju.wait(jubilant.all_active)
-    assert status.apps['testdb'].app_status.message == 'relation created'
+    assert status.apps['testdb'].units['testdb/0'].workload_status.message == 'relation created'
 
-    status2 = model2.wait(jubilant.all_active)
+    status2 = model2.wait(jubilant.all_active, timeout=900)
```

Comment:
Juju 4 just seems reliably slower in this case. 900 is a guess that seems to work.

I'm not sure whether opening a Juju ticket for a performance regression like this makes sense.